image_builder/image_builder/build.py

```diff
@@ -65,7 +65,7 @@ def build_job_image(
 
             workspace, repo_dir, git_version = prepare_workspace(workspaces_path, manifest, git_credentials, build_context, deployment_id)
 
-            if config.verify_manifest_consistency:
+            if config.verify_manifest_consistency and not build_context:
```

Comment:
For the racetrack CLI:
Should it be `and build_context==BuildContextMethod.git` or something like that?
I think the above `and not build_context` is always false for the racetrack [CLI](https://github.com/TheRacetrack/racetrack/blob/60e15ccb21da71a68dc14af6b8add1198bdc85c8/racetrack_client/racetrack_client/main.py#L62), since it selects "[BuildContextMethod.default](https://github.com/TheRacetrack/racetrack/blob/60e15ccb21da71a68dc14af6b8add1198bdc85c8/racetrack_client/racetrack_client/client/deploy.py#L44)" by default.
Not sure if the build_context is set, or not set, elsewhere in the project and might be None/False.

Reply:
In the image builder `build_context` parameter is something different. It's no longer an enum describing a type of the build context, but it's the actual compressed build context tarball, represented as bytes. It's not empty when `build_context_method == BuildContextMethod.local`, thus checking `not build_context` is enough.

Reply:
cool thanks